public/jsons/ashby_hq_aqs_builder_output.json

```diff
@@ -37,7 +37,7 @@
         ]
       },
       {
-        "attribute": "where_are_you_based_or_where_would_you_be_planning_on_workin",
+        "attribute": "where-are-you-based-or-where-would-you-be-planning-on-workin",
```

Comment:
Why hyphen rather than underscore?

Reply:
Quick fix yesterday. I need to correct it.